README.md

```diff
@@ -18,18 +18,16 @@ on our forum.
 ## DISCLAIMERS (Please read!)
 
 ### This is not a final product.
-This is **beta software**, and thus is **not stable nor is it secure**.
+This is **beta software**, and thus is potentially **unstable or insecure**.
 
 While Project Lighthouse is in a mostly working state, **we ask that our software not be used in a production
-environment until release**.
+environment without prioritizing security.**
```

Comment:
maybe provide some example of what this means (external rate limiting, ddos protection, running on a box you don't care about on a network you don't care about, not storing your banking information in an lbp level)